main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"github.com/kyokomi/emoji"
```

Comment:
this package must be in `Gopkg.toml` if you using `dep` (and it seems you do) for this task to be  completed.